LanguageFeatures/Static-extensions/static_member_A03_t01.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Consider an expression `e` which is a member invocation with
+/// syntactic receiver `C` and an associated member name `m`. Assume that `m` is
+/// a static member declared by `D`. The static analysis and dynamic semantics
+/// of this expression is the same as in Dart before the introduction of this
+/// feature.
+///
+/// @description Checks that if `C` contains static member with the basename `m`
+/// then it is not an error for extensions to declare a static members with the
+/// same basename. Invocation of `m` will invoke the appropriate member.
```

Comment:
I would expect that invocations of `m` would invoke the member declared by the `on` declaration itself, if it has a static declaration with the requested name (not just basename), and declarations in extensions would never be considered when the `on` declaration has a static member with that basename. Same approach as with instance members.